docs/working-with-gatekeeper.md

```diff
@@ -0,0 +1,2802 @@
+# Use Gatekeeper to manage OPA policy in Karmada controller plane
+
+[Gatekeeper](https://github.com/open-policy-agent/gatekeeper) , is a customizable admission webhook for Kubernetes that enforces policies executed by the Open Policy Agent (OPA), a policy engine for Cloud Native environments hosted by [Cloud Native Computing Foundation ](https://cncf.io/).
+
+This document gives an example to demonstrate how to use the `Gatekeeper` to manage OPA policy. 
+
+##Start up Karmada clusters
+You just need to clone Karmada repo, and run the following script in Karmada directory. 
+
+```
+hack/local-up-karmada.sh
+```
+
+## Start Gatekeeper
+
+In this case, we will use Gatekeeper v3.7.2. Related deployment files are from [here](https://github.com/open-policy-agent/gatekeeper/blob/release-3.7/deploy/gatekeeper.yaml).
+
+1. Create resource objects of Gatekeeper in karmada controller plane, the content is as follows. 
+
```

Comment:
If there are no additional modifications to the deployment content, I think just refer to the deployment link. Content can be brief.

Reply:
There are something different between karmada-gatekeeper-guide and what from g8sr community. Even though, I still think the draft of the guide will make users confused. Still under discussion.

Reply:
We should add some descriptions about the differences between installing Gatekeeper on `singal` cluster and `Karmada`.